node/libs/roles/src/attester/messages/l1_batch.rs

```diff
@@ -0,0 +1,122 @@
+use zksync_concurrency::time;
+
+use crate::{
+    attester::{self, AggregateSignature},
+    validator::Genesis,
+};
+
+use super::{SignedBatchMsg, Signers};
+
+#[derive(Clone, Debug, PartialEq, Eq, Hash, Default, PartialOrd)]
+/// A batch number.
+pub struct BatchNumber(pub u64);
```

Comment:
use "L1Batch" and "l1batch" names consistently (i.e. do not abbreviate to "batch").

Reply:
+1